ci/pipeline.yml

```diff
@@ -166,3 +242,10 @@ resources:
     type: slack-notification
     source:
       url: ((slack.pcap_release_webhook))
+
+  - name: blobstore
+    type: gcs
+    source:
+      bucket:   pcap-release
+      json_key: ((gcp.service_key))
+      regexp:   pcap-([0-9\.+-]+).tgz
```

Comment:
this would disallow `alpha`, etc, in case we have those in the version names anywhere.

Reply:
not true. we have successfully made a "semantic-release" version, e.g. [pcap-v1.0.0-semantic-releases.1.tgz](https://console.cloud.google.com/storage/browser/_details/pcap-release/pcap-v1.0.0-semantic-releases.1.tgz?project=app-runtime-platform-wg)

This could also be an "alpha" version, e.g. pcap-v1.0.0-alpha.1.tgz as per the [naming convention](https://github.com/semantic-release/semantic-release/blob/master/docs/support/FAQ.md#can-i-set-the-initial-release-version-of-my-package-to-001), where pre-releases are added by their branch name into the version name.

Reply:
as discussed: the regex is probably ignored somehow

Reply:
fixed regex to match alphanumeric values in the version, too.